pkgs/development/tools/stylua/default.nix

```
@@ -1,34 +1,28 @@
-{ fetchFromGitHub
-, lib
+{ lib
 , rustPlatform
-, lua52Support ? true
-, luauSupport ? false
-, fetchpatch
+, fetchFromGitHub
+, features ? [ "lua54" "luau" ]
```

Guess we could just add a comment.
```suggestion
  # lua54 implies lua52/lua53
, features ? [ "lua54" "luau" ]
```